### PR TITLE
Search enabled for devstack

### PR DESCRIPTION
**Pull request by lamagnifica**
Fixes DOC-2254. @nedbat

### CODE REVIEW

File: en_us/install_operations/source/configuration/edx_search.rst [hunk 1]
```diff
@@ -191,22 +194,23 @@ LMS
...
+  By default, all facets will be displayed. The list of available facets
+  includes:
 
   * Course organization: ``"org"``
   * Course type: ``"modes"``
   * Course language: ``"language"``
 
-* ``SEARCH_ENGINE``: Sets used search engine. There are 2 predefined values,
-  but more can be added:
+* ``SEARCH_ENGINE``: Sets used search engine. The following values are
```
Comment (nedbat) [inline en_us/install_operations/source/configuration/edx_search.rst]: Above you had "Sets the search engine to use," which is better.

File: en_us/install_operations/source/configuration/edx_search.rst [hunk 2]
```diff
@@ -154,20 +157,20 @@ The following flags are supported in the CMS and LMS applications.
...
+* ``ENABLE_COURSEWARE_INDEX``: Enables and disables courseware content and course
   info indexing.
 
-* ``ENABLE_LIBRARY_INDEX``: Enables/disables library content indexing.
+* ``ENABLE_LIBRARY_INDEX``: Enables and disables library content indexing.
 
-* ``SEARCH_ENGINE``: Sets used search engine. There are 2 predefined values,
-  but more can be added:
+* ``SEARCH_ENGINE``: Sets the search engine to use. There are two predefined
+  values, but more can be added.
```
Comment (nedbat) [inline en_us/install_operations/source/configuration/edx_search.rst]: "but more can be added" sounds like an invitation for a follow-on question, "what do you mean, and how?"  I'm not sure adding a new search engine is something within reason for the vast majority of the readers, so perhaps we should leave it out?

File: en_us/install_operations/source/configuration/edx_search.rst [hunk 3]
```diff
@@ -19,6 +19,9 @@ from within edX Platform applications. Searching is accomplished by creating an
 index of documents, and then searching within that index for matching
 information.
 
+When you install the Open edX developer stack, edX search is enabled by
+default. You must enable this feature to use it with Open edX full stack.
```
Comment (nedbat) [inline en_us/install_operations/source/configuration/edx_search.rst:23]: I didn't realize this was true.  Who can I ask about it?

Comment (lamagnifica) [inline en_us/install_operations/source/configuration/edx_search.rst:23]: looks like a solutions team change
DOC-2254 has the deets

**Comment by nedbat** [comment]
:+1:

**Comment by lamagnifica** [comment]
travis test this

**Comment by lamagnifica** [comment]
I don't know why Travis doesn't care about this PR. Builds run locally without error or warning.

**Pull request merged**